### DISCUSSION TITLE
Subfinder must root (?)

### DISCUSSION THREAD

**Discussion by kediripoesat** [projectdiscovery/subfinder#1287] [Q&A] [answered]
Aye mate, just wanna ask, if subfinder must root terminal? i already install it with kali and ubuntu, and they run on root, but no on normal terminal, so if i wanna run subfinder, katana, etc my terminal must root? sorry for my bad english :)

**Comment by ehsandeep** [comment, marked as answer]
No, it doesn't need to be root, it has something to do with user you installed with.
